lingetic-nextjs-frontend/app/components/questions/FillInTheBlanks/Question.tsx

```
@@ -50,44 +50,46 @@ export default function QuestionBox({
-          }`}
-        >
-          {isChecking ? "Checking..." : "Check"}
-        </button>
+        <div className="flex justify-end">
+          <button
+            disabled={isChecking}
+            onClick={() => {
+              void handleCheckAnswer();
+            }}
+            className="bg-[#2563eb] text-white px-6 py-3 rounded-lg font-medium hover:bg-blue-700 transition-colors flex items-center"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)